server/controllers/users.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable eqeqeq */
+
+const User = require('../models/user');
+const Document = require('../models/document');
+const Role = require('../models/role');
+
+module.exports = {
+  all: (req, res) => {
+    User.find({}, (err, users) => {
+      if (err) {
+        res.status(400).send({ error: 'Could not fetch users.' });
```

Comment:
`500`. `400` is for Bad Client requests. Since this is a server error, I think the 500 group is more appropriate.